<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObexTest</class>
 <widget class="QWidget" name="ObexTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>70</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_file">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Transferring file:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progress_file">
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bt_file_cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bt_file_done">
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>70</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bt_file_cancel</sender>
   <signal>clicked()</signal>
   <receiver>ObexTest</receiver>
   <slot>cancelClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_file_done</sender>
   <signal>clicked()</signal>
   <receiver>ObexTest</receiver>
   <slot>doneClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>cancelClicked()</slot>
  <slot>doneClicked()</slot>
  <slot>oppClicked()</slot>
  <slot>ftpClicked()</slot>
 </slots>
</ui>
